strings/isogram/isisogram.go

```diff
@@ -0,0 +1,41 @@
+// Checks if a given string is an isogram.
+// An isogram is a word, phrase, or sentence in which no letter of the alphabet occurs more than once.
+// wiki: https://en.wikipedia.org/wiki/Heterogram_(literature)#Isograms
+// Author: M3talM0nk3y
+
+package isogram
```

Comment:
Declaring a new package is unnecessary. Make this a method of the `string` package.

Reply:
@raklaptudirm Made method part of strings package. Thanks for the feedback.